test/uk/gov/hmrc/gatekeeper/services/ApplicationServiceSpec.scala

```diff
@@ -690,7 +614,7 @@ class ApplicationServiceSpec extends AsyncHmrcSpec with ResetMocksAfterEachTest
     }
 
     "propagate failure from connector" in new Setup {
-      val application = stdApp1.copy(deployedTo = Environment.SANDBOX)
+      val application = standardApp.inSandbox()
```

Comment:
Variable could be inlined. There are several others that are similarly used once only.